spec/javascripts/jasmine_spec.rb

```diff
@@ -1,16 +1,9 @@
 # frozen_string_literal: true
 require 'English'
-require 'rake'
 
 # Run the jasmine tests by running the karma javascript test framework
 # The spec will fail if any jasmine tests fails.
 RSpec.describe "Jasmine" do
-  before do
-    Rails.application.load_tasks
-    Rake::Task["assets:clobber"].invoke
```

Comment:
oh. this explains why the assets disappear in RSpec in about half the builds

Reply:
It was an ugly hack for sure.